mabot.py

```diff
@@ -0,0 +1,664 @@
+        args = [arg.replace(',', '') for arg in sub_string.split()]
+        invited = context.bot_data[update.effective_user]['UM'].current_meeting.add_persons(args)
+
+        # context.bot.send_message(chat_id=update.effective_chat.id, text=invited)
+        context.bot.send_message(chat_id=update.effective_chat.id,
+                                 text=context.bot_data[update.effective_user]['UM'].get_info(),
+                                 parse_mode='HTML')
+
+    if context.bot_data[update.effective_user]['command'] == 'remove_persons':
+        #args = [arg.replace(',', '') for arg in context.args]
+        args = [arg.replace(',', '') for arg in sub_string.split(' ')]
+        invited = context.bot_data[update.effective_user]['UM'].current_meeting.remove_persons(args)
+        # context.bot.send_message(chat_id=update.effective_chat.id, text=invited)
+        context.bot.send_message(chat_id=update.effective_chat.id,
+                                 text=context.bot_data[update.effective_user]['UM'].get_info(),
+                                 parse_mode='HTML')
+
+    if context.bot_data[update.effective_user]['command'] == 'add_options':
+        args = {i: datetime.strptime(t, '%d.%m.%Y %H:%M') for i, t in enumerate(sub_string.split(', '))}
+        options = context.bot_data[update.effective_user]['UM'].current_meeting.add_options(args)
+        # print(f'{args=}')
+        # context.bot.send_message(chat_id=update.effective_chat.id, text=options)
+        context.bot.send_message(chat_id=update.effective_chat.id,
+                                 text=context.bot_data[update.effective_user]['UM'].get_info(),
+                                 parse_mode='HTML')
+
+    if context.bot_data[update.effective_user]['command'] == 'remove_options':
+        args = {i: datetime.strptime(t, '%d.%m.%Y %H:%M') for i, t in enumerate(sub_string.split(', '))}
+        options = context.bot_data[update.effective_user]['UM'].current_meeting.remove_options(args)
+        # context.bot.send_message(chat_id=update.effective_chat.id, text=options)
+        context.bot.send_message(chat_id=update.effective_chat.id,
+                                 text=context.bot_data[update.effective_user]['UM'].get_info(),
+                                 parse_mode='HTML')
+
+    if context.bot_data[update.effective_user]['command'] == 'change_duration':
+        new_duration = int(sub_string)
+        options = context.bot_data[update.effective_user]['UM'].current_meeting.change_duration(new_duration)
+        # context.bot.send_message(chat_id=update.effective_chat.id, text=options)
+        context.bot.send_message(chat_id=update.effective_chat.id,
+                                 text=context.bot_data[update.effective_user]['UM'].get_info(),
+                                 parse_mode='HTML')
+
+
+    context.bot_data[update.effective_user]['command'] = None
+
+def main():
+
+    updater = Updater('5230717894:AAHhisY4pCyYOIUr7MkwwzYPj9TLQ7GUG5s')
```

Comment:
There is a good instrument to work with secret tokens: https://github.com/theskumar/python-dotenv

And a constant string in the middle of the code seems not to be the best solution.